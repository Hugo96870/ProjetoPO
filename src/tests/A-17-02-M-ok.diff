*** tests/A-17-02-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-17-02-M-ok.outhyp	2020-12-03 16:57:09.798569000 +0000
***************
*** 16,28 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do fornecedor: Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Registar encomenda: Operação inválida: O produto 'qwewe' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
--- 16,29 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do fornecedor: Identificador do fornecedor: Identificador do produto: Quantidade: Quantidade: Quantidade: Registar encomenda: Operação inválida: O produto '2' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
