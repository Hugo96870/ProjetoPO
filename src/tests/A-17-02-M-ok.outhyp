Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do fornecedor: Identificador do fornecedor: Identificador do produto: Quantidade: Quantidade: Quantidade: Registar encomenda: Operação inválida: O produto '2' não existe.
Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Número inválido!
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Fornecedores
1 - Mostrar fornecedores
2 - Registar
3 - Permitir/inibir transacções
4 - Consultar transacções
0 - Sair
Escolha uma opção: Identificador do fornecedor: Gestão de Fornecedores
1 - Mostrar fornecedores
2 - Registar
3 - Permitir/inibir transacções
4 - Consultar transacções
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 