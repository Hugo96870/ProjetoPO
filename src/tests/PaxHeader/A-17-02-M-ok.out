30 mtime=1606841711.255074817
